next.config.mjs

```diff
@@ -27,8 +27,27 @@ const nextConfig = {
   // We allow the BASE_PATH to be overridden in case that the Website
   // is being built on a subdirectory (e.g. /nodejs-website)
   basePath: BASE_PATH,
-  // We disable image optimisation during static export builds
-  images: { unoptimized: ENABLE_STATIC_EXPORT },
+  images: {
+    // We disable image optimisation during static export builds
+    unoptimized: ENABLE_STATIC_EXPORT,
+    // We allow SVGs to be used as images
+    dangerouslyAllowSVG: true,
```

Comment:
@styfle  For example, we were getting error when we were not able to load the following image on the https://nodejs.org/en/about/previous-releases page

The image we are trying to load;
https://raw.githubusercontent.com/nodejs/Release/main/schedule.svg?sanitize=true

```bash
The requested resource "https://raw.githubusercontent.com/nodejs/Release/main/schedule.svg?sanitize=true" has type "image/svg+xml" but dangerouslyAllowSVG is disabled
```

Reply:
@canerakdas Seem like you could add unoptimized=true prop for that image instead of opening the risk for all images with this config.

Reply:
As far as I know, since this image is generated from Markdown using MDX, I can't directly add `unoptimized`. prop Instead, I updated the `MDXImage` component to serve images with the `.svg` extension as `unoptimized`

Here is the PR; https://github.com/nodejs/nodejs.org/pull/7244

Thank you for the feedback!